### DISCUSSION TITLE
repeating keys

### DISCUSSION THREAD

**Discussion by klue-less** [isontheline/pro.webssh.net#359] [Q&A]
Is there a way to configure terminals' keys to repeat when held down on a non-hardware keyboard?

**Comment by isontheline** [comment]
Hey @klue-less 👋

No need to [flood](https://github.com/isontheline/pro.webssh.net/discussions/358) : I have created an issue to track progress : https://github.com/isontheline/pro.webssh.net/issues/361

Yet there is no way to configure keys repeat.
I'm searching the iOS documentation about this behavior.
